src/project/project-actions.tsx

```diff
@@ -259,7 +291,7 @@ export class ProjectActions {
    * @param files the files from drag and drop or an input element.
    * @param the type of user event that triggered the load.
    */
-  load = async (
+  loadHex = async (
```

Comment:
Not the best naming, as this function can handle more than just hex files, though it is its main purpose.